quickstarts/microsoft.azurestackhci/vm-windows-disks-and-adjoin/prereqs/logical-network/main.bicep

```
@@ -1,47 +1,65 @@
-@description('The name of an existing Logical Network in your HCI cluster - for example: vnet-compute-vlan240-dhcp')
+@description('The name of an logical network - for example: vnet-compute-vlan240-dhcp')
 param logicalNetworkName string
+
 @description('The name of a Hyper-V VM switch in your HCI cluster - usually serving your Compute network. For example: ComputeSwitch(compute)')
 param vmSwitchName string
 @description('The DNS servers to use for the logical network. Make sure to use local DNS servers for AD-joined systems')
-param dnsServers array = [
-  '1.1.1.1'
-  '8.8.8.8'
-]
+param dnsServers array = []
 @description('The VLAN ID for the logical network. If not specified, the default value is 0.')
 param vlan int = 0
 @description('The address prefix for the logical network - for example: 172.16.0.0/22')
-param addressPrefix string
+param addressPrefix string = ''
```

Make this an optional parameter with a default null value?

```suggestion
param addressPrefix string?
```